architecture/topics/aws_getting_started_install.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-install-{context}']
+= Installing the AWS Service Broker into an existing {product-title} cluster
+
+To install the AWS broker into your {product-title} cluster:
+
+. Create a directory for the AWS Service Broker installation files:
++
+----
+mkdir awssb
+cd awssb
+----
+
+. Download the installation files:
++
+----
+wget https://raw.githubusercontent.com/awslabs/aws-servicebroker/release-v1.0.0-beta.2/packaging/openshift/deploy.sh
+wget https://raw.githubusercontent.com/awslabs/aws-servicebroker/release-v1.0.0-beta.2/packaging/openshift/aws-servicebroker.yaml
+wget https://raw.githubusercontent.com/awslabs/aws-servicebroker/release-v1.0.0-beta.2/packaging/openshift/parameters.env
+chmod +x deploy.sh
+----
++
+[NOTE]
+====
+AWS is responsible for any issues involving these scripts.
+====
+
+. Edit the `parameters.env` as needed:
++
+You can leave the default values for most installations. 
++
+----
+TARGETACCOUNTID= <1>
+TARGETROLENAME=  <1>
+VPCID=  
+REGION=us-east-1
+IMAGE=awsservicebroker/aws-servicebroker:beta
+IMAGEPULLPOLICY=Always
+S3BUCKET=awsservicebroker
+S3KEY=templates/latest
+S3REGION=us-east-1
+TABLENAME=awssb
+VERBOSITY=10
+BROKERID=awsservicebroker
+PRESCRIBE_OVERRIDES=true
+----
++
+<1> Leave these parameters blank if you are installing the AWS Service Broker using the account where the user or role is located. 
+If you are using a different AWS account, specify the account ID and role for that account.
+
+. Run the following command to start the installation script:
++
+----
+./deploy.sh <1>
+----
++
+<1> If you are using a different AWS account, specify the account ID and secret key for that account:
++
+----
+./deploy.sh <ACCESS_KEY_ID> <SECRET_KEY>
```

Comment:
Need a $
I'd probably just show the command once and mention when you can omit the parameters.